From: Sven Eckelmann <sven@narfation.org>
Date: Sat, 3 Dec 2016 14:26:34 +0100
Subject: alfred: upgrade package to latest release 2016.5

 * support interface validity checks on systems without debugfs
 * remove debugfs check during batadv-vis startup
 * allow out-of-order txend packets during transmissions

Signed-off-by: Sven Eckelmann <sven@narfation.org>

Origin: backport, https://github.com/openwrt-routing/packages/commit/e18c440d8b062603477e49a67ea9de31f749abce

diff --git a/alfred/Makefile b/alfred/Makefile
index 09f62d4..b5d37ce 100644
--- a/alfred/Makefile
+++ b/alfred/Makefile
@@ -11,9 +11,9 @@ include $(TOPDIR)/rules.mk
 # The latest alfred git hash in PKG_REV can be obtained from https://git.open-mesh.org/alfred.git
 #
 PKG_NAME:=alfred
-PKG_VERSION:=2016.4
+PKG_VERSION:=2016.5
 PKG_RELEASE:=0
-PKG_MD5SUM:=172501ca734ac584fa13f10dcfa708da
+PKG_MD5SUM:=e03d422ed3b5a162b90e8af13389523f
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
